Controller/Gateway/MoneroPayment.php

```diff
@@ -399,120 +474,145 @@ public function execute()
         $country = '';
         $region = '';
         $paramCount = 0;
-        
-        if(isset($_GET['first'])){
-            $first = $_GET['first'];
+        $errors = [];
+
+        if (isset($_GET['first'])) {
```

Comment:
Just noticing this, why `$_GET`? Are we sending everything as a GET? I would prefer a POST 

> Besides keeping the URL relatively cleaner, POST also lets you send much more information (as URLs are limited in length, for all practical purposes), and lets you send just about any type of data (file upload forms, for example, can't use GET -- they have to use POST plus a special content type/encoding).

Reply:
Better `first` ---> `first_name`

Reply:
The line `if (isset($_GET['first'])) {` checks if the 'first' parameter is present in the URL query string. $_GET is a [superglobal array in PHP](https://www.php.net/manual/en/reserved.variables.get.php) that contains data sent via the URL. I'm not the original author of the code but I believe the shortened parameter version like 'first' was used intentionally as the URL length has some limitations.